CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 0.12.0 (April 22, 2024)
+
+NOTES:
+* **This release requires juju controller version 3.4 or later juju.**
```

Comment:
It requires 2.9.49 or later for general content. To use with JAAS login enhancements, requires 3.5.0+. To use with juju secrets requires 3.4.0+